docs/components/novabotflow.md

```diff
@@ -8,6 +8,48 @@
 #Nova.BotFlow
 ##Product Notices
 ***
+
+###2022R4-5 :briefcase: Epic
+####New Features
+- *Nova.Botflow*: Add "Video" processing (FB Messenger) [:clipboard: NC2-177] [NC2-177]
+
+	- Added processing of outgoing videos to the code .\src\node-red-contrib-chatbot\lib\platforms\facebook\facebook.js
+	> .\src\node-red-contrib-chatbot\lib\platforms\facebook\facebook.js
+
+- *Nova.Botflow* IVR menu: chatBot changes [:clipboard: NC2-368] [NC2-368]
+	- Updated botflow according to new [Menu (IVR) TK](https://drive.google.com/open?id=1wl5w_mNff59Cdrz0BJfL9iI3zrFnDCjw&authuser=cristina.podoliuh%40novait.com.ua&usp=drive_fs).
+
```

Comment:
19: broken format
20: list of updates and only then link to the "specification", not "TK" + check parametrization - link is wrong